### PR TITLE
init renv

### PR DESCRIPTION
**Pull request by alexvpickering**
# Background
#### Link to issue 
https://biomage.atlassian.net/secure/RapidBoard.jspa?rapidView=1&projectKey=BIOMAGE&modal=detail&selectedIssue=BIOMAGE-1132
#### Link to staging deployment URL 

https://ui-olivergibson-pipeline110.scp-staging.biomage.net/
#### Anything else the reviewers should know about the changes here

# Changes
### Code changes

# Definition of DONE
Your changes will be ready for merging after each of the steps below have been completed:

### Testing
- [ ] Unit tests written
- [x] Tested locally with Inframock (with latest production data downloaded with biomage experiment pull)
- [x] Deployed to staging

To set up easy local testing with inframock, follow the instructions here: https://github.com/biomage-ltd/inframock
To deploy to the staging environment, follow the instructions here: https://github.com/biomage-ltd/biomage-utils

### Documentation updates
Is all relevant documentation updated to reflect the proposed changes in this PR?

- [ ] Relevant Github READMEs updated
- [ ] Relevant wiki pages created/updated

### Approvers
- [ ] Approved by a member of the core engineering team
- [ ] (UX changes) Approved by vickymorrison (this is her username, tag her if you need approval)

### Just before merging:
- [ ] After the PR is approved, the `unstage` script in here: https://github.com/biomage-ltd/biomage-utils is executed. This script cleans up your deployment to staging

### Optional
- [ ] Photo of a cute animal attached to this PR

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,16 +0,0 @@
-version: "3.2"
```
Comment (ogibson-biomage) [inline docker-compose.yml:1]: My only question for this PR is why we no longer need the docker-compose file

Comment (alexvpickering) [reply docker-compose.yml:1]: It hasn't been used since @marcellp migrated the repo away from `remoter`.  Just a bit of housekeeping

Comment (alexvpickering) [reply docker-compose.yml]: Just in case I tried restoring this but it did not resolve the issue we are seeing in staging

Comment (ogibson-biomage) [reply docker-compose.yml]: I'll take a look today and try to fix it.

File: pipeline-runner/renv.lock.init [hunk 2]
```diff
@@ -0,0 +1,1623 @@
...
+      "Package": "paws.application.integration",
+      "Version": "0.1.11",
+      "Source": "Repository",
+      "Repository": "CRAN",
+      "Hash": "15be816c36116393eff157fe061cb564"
+    },
+    "paws.common": {
+      "Package": "paws.common",
+      "Version": "0.3.10",
+      "Source": "Repository",
```
Comment (ogibson-biomage) [inline pipeline-runner/renv.lock.init:1048]:
```suggestion
      "Version": "0.3.11",
      "Source": "Repository",
```

File: pipeline-runner/renv.lock [hunk 3]
```diff
@@ -0,0 +1,1623 @@
...
+      "Package": "paws.application.integration",
+      "Version": "0.1.11",
+      "Source": "Repository",
+      "Repository": "CRAN",
+      "Hash": "15be816c36116393eff157fe061cb564"
+    },
+    "paws.common": {
+      "Package": "paws.common",
+      "Version": "0.3.10",
+      "Source": "Repository",
```
Comment (ogibson-biomage) [inline pipeline-runner/renv.lock:1048]:
```suggestion
      "Version": "0.3.11",
      "Source": "Repository",
```

**Review by ogibson-biomage — APPROVED** [review]

**Review by ogibson-biomage — APPROVED** [review]

**Pull request merged**